st2api/st2api/controllers/v1/webhooks.py

```diff
@@ -51,11 +51,12 @@ def __init__(self, *args, **kwargs):
         self._trigger_types = WEBHOOK_TRIGGER_TYPES.keys()
 
         self._trigger_dispatcher = TriggerDispatcher(LOG)
+        queue_suffix = self.__class__.__name__ + ':' + str(uuid.uuid4())
         self._trigger_watcher = TriggerWatcher(create_handler=self._handle_create_trigger,
                                                update_handler=self._handle_update_trigger,
                                                delete_handler=self._handle_delete_trigger,
                                                trigger_types=self._trigger_types,
-                                               queue_suffix=self.__class__.__name__,
```

Comment:
Good catch :+1: 

We still use a topic exchange here so the messages should be delivered in round-robin fashion so that's fine, right?